src/transformers/models/bridgetower/image_processing_bridgetower.py

```diff
@@ -155,10 +156,13 @@ class BridgeTowerImageProcessor(BaseImageProcessor):
             Standard deviation to use if normalizing the image. This is a float or list of floats the length of the
             number of channels in the image. Can be overridden by the `image_std` parameter in the `preprocess` method.
             Can be overridden by the `image_std` parameter in the `preprocess` method.
-        do_center_crop (`bool`, *optional*, defaults to `True`):
+        do_center_crop (`bool`, defaults to `True`, *optional*, defaults to `True`):
             Whether to center crop the image. Can be overridden by the `do_center_crop` parameter in the `preprocess`
             method.
-        do_pad (`bool`, *optional*, defaults to `True`):
+        crop_size (`Dict[str, int]`, defaults to `None`, *optional*):
```

Comment:
The library pattern is to not say any type if the default is `None` 

```suggestion
        crop_size (`Dict[str, int]`, *optional*):
```